java/code/src/com/suse/manager/webui/websocket/WebsocketSessionConfigurator.java

```diff
@@ -26,11 +28,20 @@
  * WebsocketSessionConfigurator
  */
 public class WebsocketSessionConfigurator extends ServerEndpointConfig.Configurator {
+
+    private static final Logger LOG = Logger.getLogger(WebsocketSessionConfigurator.class);
+
     @Override
     public void modifyHandshake(ServerEndpointConfig config,
             HandshakeRequest request,
             HandshakeResponse response) {
         HttpSession httpSession = (HttpSession)request.getHttpSession();
-        config.getUserProperties().put("webUserID", httpSession.getAttribute("webUserID"));
+        if (httpSession.getAttribute("webUserID") != null) {
+            config.getUserProperties().put("webUserID", httpSession.getAttribute("webUserID"));
+        }
+        else {
+            LOG.info("session field 'webUserID' was null during websocket handshake");
```

Comment:
Shouldn't that be a LOG.error?

Reply:
We decided to keep it as info, simply because it's normal for this to happen and we don't want to annoy/alert users, but still want to keep an eye on it when it happens